CMakeLists.txt

```diff
@@ -187,19 +187,42 @@ if(BUILD_CLIENT)
         find_package(Sparkle)
     endif()
 
-    if(UNIX)
-        find_package(INotify REQUIRED)
-    else()
-        find_package(INotify)
+    if (NOT APPLE)
+        if(UNIX)
+            find_package(INotify REQUIRED)
+        else()
+            find_package(INotify)
+        endif()
+        find_package(Sphinx)
+        find_package(Libcloudproviders)
     endif()
-    find_package(Sphinx)
     find_package(PdfLatex)
+
+    if (APPLE)
+        find_program(HOMEBREW_EXECUTABLE brew)
+        mark_as_advanced(FORCE HOMEBREW_EXECUTABLE)
+        if (HOMEBREW_EXECUTABLE)
```

Comment:
Wondering, isn't it something we'd want to do a bit earlier in the game? After all we're having a couple of other find_package calls earlier, and I expect others might be introduced in the future which could be provided by homebrew as well.

Reply:
> On Windows for instance I'm using chocolatey, so should we start adding code to accomodate it? And what when something else comes around on Mac which installs in some different non standard places?

Homebrew is explicitly part of the install instructions for Mac. At the very least it would make sense to add the Homebrew prefix to CMAKE_PREFIX_PATH at some point along the line (inside of an if(APPLE) block).